charts/rundeck/templates/rundeck-backend-deployment.yaml

```diff
@@ -129,15 +129,15 @@ spec:
               path: /
               port: 4440
               scheme: HTTP
-            initialDelaySeconds: 120
-            periodSeconds: 5
+            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
           readinessProbe:
             httpGet:
               path: /
               port: 4440
               scheme: HTTP
-            initialDelaySeconds: 60
-            periodSeconds: 5
+            initialDelaySeconds: {{ .Values.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.readinessProbe.periodSeconds }}
```

Comment:
could you please make the fields mandatory ( like https://github.com/EugenMayer/helm-charts/blob/main/charts/rundeck/templates/rundeck-backend-deployment.yaml#L78 ) since the entire chart would not start if those values are missing.